optimize-nextflow.py

```diff
@@ -216,17 +220,41 @@ def generate(self) -> str:
         Returns:
             Serialized Nextflow configuration.
         """
-        prefix = "process { \n"
-        suffix = "\n }"
-
         process_configs = list()
         for process in self.process_metrics:
             process_config = self.generate_single(process)
             process_config_fmt = indent(process_config, "  ")
             process_configs.append(process_config_fmt)
         process_configs_concat = "".join(process_configs)
+        process_configs_concat = indent(process_configs_concat, " " * 8)
+
+        full_config = f"""\
```

Comment:
Nit: Does is make sense to use jinja templates for these kind of things?

Reply:
Now that this template has grown in complexity, I agree that we should look into switching to Jinja going forward. I'm going to leave a TODO for now. We can do the switch when we incorporate this code into py-orca.